src/UserService.Models.Db/DbUser.cs

```diff
@@ -60,6 +63,10 @@ public void Configure(EntityTypeBuilder<DbUser> builder)
                 .Property(p => p.LastName)
                 .IsRequired();
 
+            builder
```

Comment:
int can't be null, it is not requered to write this

Reply:
done